flake.nix

```diff
@@ -27,12 +27,23 @@
           devShells.default = pkgs.fromTOML ./devshell.toml;
         }
       ) // {
-      templates.default.path = ./template;
-      templates.default.description = "nix flake new 'github:numtide/devshell'";
+
+      templates = rec {
+        toml = {
+          path = ./templates/toml;
+          description = "nix flake new my-project -t github:numtide/devshell";
+        };
+        flake-parts = {
+          path = ./templates/flake-parts;
+          description = "nix flake new my-project -t github:numtide/devshell#flake-parts";
+        };
+        default = toml;
```

Comment:
@zimbatm I added a template as well, keeping the original toml based one as the default.